pkg/prompts/prompts.go

```diff
@@ -23,23 +23,39 @@ import (
 const (
 	Yes = "Yes"
 	No  = "No"
+
+	Add      = "Add"
+	Del      = "Delete"
+	Preview  = "Preview"
+	MoreInfo = "More Info"
+	Done     = "Done"
+	Cancel   = "Cancel"
 )
 
 type Prompter interface {
 	CapturePositiveBigInt(promptStr string) (*big.Int, error)
-	CaptureAddress(promptStr string) (common.Address, error)
+	CaptureAddress(promptStr string, arg any) (any, error)
 	CaptureExistingFilepath(promptStr string) (string, error)
 	CaptureYesNo(promptStr string) (bool, error)
 	CaptureNoYes(promptStr string) (bool, error)
 	CaptureList(promptStr string, options []string) (string, error)
 	CaptureString(promptStr string) (string, error)
-	CaptureIndex(promptStr string, options []common.Address) (int, error)
+	CaptureIndex(promptStr string, options []interface{}) (int, error)
 	CaptureDuration(promptStr string) (time.Duration, error)
 	CaptureDate(promptStr string) (time.Time, error)
 	CaptureNodeID(promptStr string) (ids.NodeID, error)
 	CaptureWeight(promptStr string) (uint64, error)
 	CaptureUint64(promptStr string) (uint64, error)
-	CapturePChainAddress(promptStr string, network models.Network) (string, error)
+	CapturePChainAddress(promptStr string, network any) (any, error)
+	CaptureListDecision(
```

Comment:
more descriptive names for params, and comments

Reply:
Added comments just as the other place.